### DISCUSSION TITLE
how to handle require-description in splitted files for Query/Mutation/Subscription

### DISCUSSION THREAD

**Discussion by gwildu** [graphql-hive/graphql-eslint#1122] [Ideas]
In our project, we have splitted schema files and therefore duplicated query, mutation and subscription types. I think this is a common practice. We are because of this struggling a bit with the require-description rule, as it requires description for every query, mutation, subscription type in every file. 

This behavior was introduced with https://github.com/B2o5T/graphql-eslint/issues/944 

I tried to change de default options of the rule like this but that does not solve the issue:
```
"@graphql-eslint/require-description": [
  "error",
    {
      types: true,
      DirectiveDefinition: true,
      OperationDefinition: false,
    },
],
```
Is there already a solution for that? If not maybe enabling the above configuration might be a solution.

**Comment by dimaMachina** [comment]
Hi, can you create a reproduction repo of your problem ?

**Reply by dimaMachina** [reply]
Because I guess something incorrect in your configuration

**Reply by gwildu** [reply]
Thanks for the feedback, I will try to extract a repo from our project, but it could take a while as for the next 2 weeks I'm in vacation...
